.github/workflows/build_and_push.yml

```diff
@@ -10,6 +9,7 @@ on:
 jobs:
   build_images:
     runs-on: ubuntu-latest
+    if: ${{ github.repository_owner	== 'trezor' }}
```

Comment:
There seems to be a lot of whitespace at the left of `==`. Same in the file below